src/main/resources/default/taglib/k/section.html.pasta

```diff
@@ -14,7 +14,7 @@
     <div class="card-body">
         <div class="d-flex justify-content-between @if (!isFilled(heading)) { flex-row-reverse }">
             <i:if test="isFilled(heading)">
-                <h5 id="@anchor" class="card-title @headingClass">
+                <h5 id="@anchor" class="card-title anchor-scroll-margin @headingClass">
```

Comment:
if we need a style only in a single place, it can also be but right at the element as there is no benefit in using a separate css class...